ibis/backends/tests/test_struct.py

```diff
@@ -23,6 +24,38 @@
 ]
 
 
+@pytest.mark.notimpl(["postgres"])
+# @pytest.mark.broken(["pandas", "dask"], reason="casting is broken")
```

Comment:
```suggestion
```

Reply:
here is the test case that exercises the change in formats/pandas.py.

Reply:
no longer relevant.